### PR TITLE
(improvement) Tables: removed column header 'actions'

### PR DESCRIPTION
**Pull request by dmytroshch**
tickets: 
https://app.asana.com/0/1200372033300327/1200621960286255 
https://app.asana.com/0/1200372033300327/1200590321754787

Removed sorting and header title

Снимок экрана от 2021-07-20 12-10-23
Снимок экрана от 2021-07-20 12-10-26
Снимок экрана от 2021-07-20 12-10-30

### CODE REVIEW

**Review by JasonMHasperhoven — APPROVED** [review]

File: src/components/AtomicOrdersTable/AtomicOrdersTable.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ const AtomicOrdersTable = ({
         <VirtualTable
           data={orders}
           columns={AtomicOrdersTableColumns(authToken, cancelOrder, gaCancelOrder, size)}
-          defaultSortBy='id'
```
Comment (bhoomij) [inline src/components/AtomicOrdersTable/AtomicOrdersTable.js:23]: I think earlier it was sorting based on id and so latest placed orders where on top, and with this change they will be sorted by name. 
Could you confirm with Ivan/Team if we want to change the sorting behaviour ?

**Review by bhoomij — COMMENTED** [review]
looks good, please check if we want to change sorting from id to symbol

File: src/components/AtomicOrdersTable/AtomicOrdersTable.js (hunk 1 above)
Comment (dmytroshch) [reply src/components/AtomicOrdersTable/AtomicOrdersTable.js:23]: @bhoomij we don't have column keys "id" in AtomicOrdersTable's column's

Comment (dmytroshch) [reply src/components/AtomicOrdersTable/AtomicOrdersTable.js:23]: also in PositionTable column 'actions' has id 'key', which is wrong

Comment (Blaumaus) [reply src/components/AtomicOrdersTable/AtomicOrdersTable.js:23]: @dmytroshch orders have an id field, although it is not visually displayed in the table
this field is used to sort orders by their creation date

File: src/components/AlgoOrdersTable/AlgoOrdersTable.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ const AlgoOrdersTable = ({
       <VirtualTable
         data={orders}
         columns={AlgoOrdersTableColumns(authToken, cancelOrder, gaCancelOrder)}
-        defaultSortBy='gid'
+        defaultSortBy='id'
```
Comment (bhoomij) [inline src/components/AlgoOrdersTable/AlgoOrdersTable.js:19]: we are changing from gid to id, is that intended change and will it still sort by creation date ?

File: src/components/PositionsTable/PositionsTable.columns.js [hunk 3]
```diff
@@ -56,8 +56,7 @@ export default ({ authToken, closePosition }) => [{
   flexGrow: 1,
   cellRenderer: ({ rowData = {} }) => processBalance(preparePrice(rowData.marginFunding)),
 }, {
-  label: 'Actions',
-  dataKey: 'id',
+  dataKey: 'cid',
```
Comment (bhoomij) [inline src/components/PositionsTable/PositionsTable.columns.js:59]: just wanted to make sure that change in dataKey from `id -> cid` is not accidental ? 
Delete action will still work with `cid` ?

Comment (Blaumaus) [reply src/components/PositionsTable/PositionsTable.columns.js:59]: Positions don't have a `cid`, either `id` or `order_cid` / `order_id` from `meta` should be used

**Review by bhoomij — APPROVED** [review]

**Pull request merged**